### ISSUE TITLE
Mobile menu accessibility 

### ISSUE THREAD

**Issue by andrewremery** [andrewremery/ithaca_test#23]
Look into if an[ aria-live ](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live)attribute should be added to the menu when it is opened on mobile. 

If it is needed it will most likely need to be doggled in[ toggleMobileMenu function](https://github.com/andrewremery/ithaca_test/blob/2c2247f50953bca08e555504caf712a00595e791/js/scripts.js#L6-L8)
